_posts/0000-01-08-bituin-1mwtt.md

```diff
@@ -2,5 +2,6 @@
 layout: slide
```

Comment:
Dear @bituin please add back the missing file `https://raw.githubusercontent.com/ilike-1mwtt/github-slideshow/master/_posts/0000-01-02-ilike-1mwtt.md` as `0000-01-02-ilike-1mwtt.md` and rename your file to `0000-01-06-bituin-1mwtt.md` and commit.